templates/carecoordination/cda/cda-validate-results.json.twig

```diff
@@ -0,0 +1,4 @@
+{
+    'document': {{ document.get_id() }}
```

Comment:
would pipe this through js_escape

Reply:
or actually better to do through json_encode

Reply:
get_id returns an integer right?  I don't want this to be a string.

Reply:
json_encode is smart and won't put quotes around integers (can try it out at http://php.fnlist.com/php/json_encode)

Reply:
k, sounds good.  For some reason I though it always string encoded.

Reply:
to be safe you may want to force it to an int prior to the call since it will treat '1' as string and 1 as int

Reply:
I'll just leave it alone.